spec/amp-framework-hosting.md

```diff
@@ -0,0 +1,234 @@
+# Hosting the AMP framework
+
+You can host the AMP framework and components from your own server or CDN. This feature has a number of applications. For example, you can...
+
+- test and demonstrate changes to the framework or components.
```

Comment:
It does. An important bullet point in offering publishers the ability to host the AMP framework is _control_. This is possible since we've abandoned the idea of mixing a self-hosted AMP framework with certain components from `cdn.ampproject.org`. FWIW, this is possible today. The only feature under review right now is `<meta name="runtime-host">` support. Publishers can opt to build the AMP framework today, specifying a different host in `AMP_CONFIG.urls.cdn`, and make all the modifications they like. This just documents that feature.